src/components/notifications/response-notification.jsx

```diff
@@ -1,12 +1,12 @@
 import { InlineNotification } from "carbon-components-react";
 import React from "react";
 
-export const errorNotification = (message) => {
+export const ResponseNotification = (kind, title, message) => {
```

Comment:
can we follow camel case format everywhere.. since this is common function which just renders notification, having 'R' will confuse people that this is a component